packages/experiments/src/components/Toggle/Toggle.types.ts

```diff
@@ -88,39 +99,46 @@ export type IToggleViewProps = Pick<IToggleProps, 'as' | 'label' | 'ariaLabel' |
      * Text to display next to the toggle.
      */
     text?: string;
+
+    /**
+     * Reference to the toggle button.
+     */
+    toggleButton?: React.RefObject<HTMLButtonElement>;
   };
 
 /**
  * Styles for the Toggle component.
  */
-export interface IToggleStyles {
-  /**
-   * Root element.
-   */
-  root: IStyle;
-
-  /**
-   * Label element above the toggle.
-   */
-  label: IStyle;
-
-  /**
-   * Container for the toggle pill and the text next to it.
-   */
-  container: IStyle;
-
-  /**
-   * Pill, rendered as a button.
-   */
-  pill: IStyle;
+export interface IToggleStyleVariablesTypes {
+  pillBackground?: string;
+  pillHoveredBackground?: string;
+  pillBorderColor?: string;
+  pillHoveredBorderColor?: string;
+  pillJustifyContent?:
+    | 'initial'
+    | 'inherit'
+    | 'unset'
+    | 'flex-start'
+    | 'flex-end'
+    | 'center'
+    | 'space-between'
+    | 'space-around'
+    | 'space-evenly'
+    | 'stretch'
+    | undefined;
+  pillHighContrastBackground?: string;
+  pillHighContrastHoveredBackground?: string;
+  pillHighContrastBorderColor?: string;
+  pillHighContrastHoveredBorderColor?: string;
+
+  thumbBackground?: string;
+  thumbHighContrastBackground?: string;
+  thumbHighContrastBorderColor?: string;
+
+  textColor?: string;
+  textHighContrastColor?: string;
+}
 
-  /**
-   * Thumb inside of the pill.
-   */
-  thumb: IStyle;
+export type IToggleStyleVariables = { [PState in IToggleStates]?: IToggleStyleVariablesTypes };
 
-  /**
-   * Text next to the pill.
-   */
-  text: IStyle;
-}
```

Comment:
We need to work out with this and Button and with @dzearing whether we keep these interfaces for backwards compatibility or roll out as a breaking change with Fabric 7